pages/api/v1/discord/event.ts

```diff
@@ -32,6 +32,15 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse<
   if (new Date(end) < new Date())
     return res.status(400).json({ error: "End date cannot be in the past" });
 
+  if(image){
+  // Discord only accepts gif/jpeg/png for cover images on scheduled events
+    const discordImageTypes = ['gif', 'jpeg', 'jpg', 'png'];
+    const extRe = new RegExp("\.([0-9a-z]+)(?:[\?#]|$)", "i");
+    const ext = extRe.exec(image);
```

Comment:
If we're using this both here and in util/discord.ts, it makes more sense to have this be its own function in util/discord.ts